src/sections/Main/Main.jsx

```diff
@@ -5,7 +5,7 @@ export default function Main() {
   return (
     <header className={classes.header}>
       <div className={classes.top}>
-        <img src="/img/logo.svg" />
+        <img src="/img/logo.svg" width="40" height="40" alt="logo" />
```

Comment:
I'd handle that in css and if you want to be lighthouse compliant wrap it with container +

```css
img {
  height: auto;
  width: 100%;
}
```

Reply:
"Image elements do not have explicit width and height
Set an explicit width and height on image elements to reduce layout shifts and improve CLS. "

Reply:
indeed and if you read a little further, you'll see snippet I've attached https://web.dev/optimize-cls/?utm_source=lighthouse&utm_medium=devtools#modern-best-practice